code-challenges.test.js

```diff
@@ -14,33 +14,131 @@
+// Snapshots:   0 total
+
+
 
 // --------------------2) Create a function that takes in a mixed data array and returns an array of only the REMAINDERS of the numbers when divided by 3.
 
+
 // a) Create a test with an expect statement using the variables provided.
+describe("getRemainders", () => {
+test("returns an array of the remainders of the numbers when divided by 3", () => {
+  const hodgepodge1 = [23, "Heyyyy!", 45, -10, 0, "Yo", false]
+  const hodgepodge2 = [5, "Hola", 43, -34, "greetings", true]
+
+  expect(getRemainders(hodgepodge1)).toEqual([2, 0, -1, 0])
+  expect(getRemainders(hodgepodge2)).toEqual([2, 1, -1])
+})
+})
 
-const hodgepodge1 = [23, "Heyyyy!", 45, -10, 0, "Yo", false]
-// Expected output: [ 2, 0, -1, 0 ]
-const hodgepodge2 = [5, "Hola", 43, -34, "greetings", true]
-// Expected output: [ 2, 1, -1 ]
 
 // b) Create the function that makes the test pass.
 
+//pseudocode
+// declare a function remainders
+// we want filter out the numbers from the array 
+// we wnat to check to make sure it is a number then filter it out into a new array 
+// once its filtered out were going to take that number and make a new array of the remainders
+// were going to return the remainder
+
+getRemainders = (arr) => {
+  return arr.filter(elem => typeof elem === 'number').map(num => num % 3)
+}
+
+// Test Suites: 1 passed, 1 total
+// Tests:       2 passed, 2 total
+// Snapshots:   0 total
+// Time:        0.454 s, estimated 1 s
+// Ran all test suites.
+
 // --------------------3) Create a function that takes in an array of numbers and returns the sum of all the numbers cubed.
 
 // a) Create a test with an expect statement using the variables provided.
 
+describe("sumCubed", () => {
+  const cubeAndSum1 = [2, 3, 4]
+  const cubeAndSum2 = [0, 5, 10]
+test("should return the sum of all the numbers cubed", () => {
+  expect(sumCubed(cubeAndSum1)).toEqual(99)
+  expect(sumCubed(cubeAndSum2)).toEqual(1125)
+})
+})
 const cubeAndSum1 = [2, 3, 4]
 // Expected output: 99
 const cubeAndSum2 = [0, 5, 10]
 // Expected output: 1125
 
 // b) Create the function that makes the test pass.
+
+// declare a function
+// were going to take an array as the paramater
+// declare a new constant for the sum make that 0
+// iterate through each number in the array and multiply by 3
+// add that to the sum we declared earlier
+// return the sum
+
+// sumCubed = (arr) => {
+//   let sum = 0;
+//   for (let i = 0; i < arr.length; i++) {
+//     sum += (arr[i] * 3)
+//   }
+//   return sum
+// }
+
+// wrote this code here didnt work i was expecting 99 i got 27
+// I was just multiplying by three
+// I used math.pow to cube the number and then add it to the sum 
+
+sumCubed = (arr) => {
+  let sum = 0;
+  for (let i = 0; i < arr.length; i++) {
+    sum += Math.pow(arr[i], 3)
+  }
+  return sum
+}
+
+// Test Suites: 1 passed, 1 total
+// Tests:       3 passed, 3 total
+// Snapshots:   0 total
+// Time:        0.464 s, estimated 1 s
```

Comment:
Way to find a method to help you.  Great solution!  To step it up a little, try using a higher order function instead of a for loop.  This works, and it's a great way to do it, but that would just move it up a level.  Awesome job!